ImageNet/train/dawn_resnet50_tpu_tensorflow17.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Can you also rename this file (and the other one) to something like `google_resnet50_tpu_tensorflow17.json` -- the first term is supposed to represent the organization making the submission?